src/users/users.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Controller, Get, Post, Patch, Param, Delete } from '@nestjs/common';
+import { UsersService } from './users.service';
+
+@Controller()
+export class UsersController {
+  constructor(private readonly usersService: UsersService) {}
+
+  @Post()
+  create() {
+    return this.usersService.create();
+  }
+
+  @Get()
+  findAll() {
+    return this.usersService.findAll();
+  }
+
+  @Get(':id')
+  findOne(@Param('id') id: string) {
+    return this.usersService.findOne(id);
+  }
+
+  @Patch(':id')
+  update(@Param('id') id: string) {
+    return this.usersService.update(id);
+  }
+
+  @Delete(':id')
+  remove(@Param('id') id: string) {
+    return this.usersService.remove(id);
+  }
```

Comment:
I know that this is how nest generates CRUD, but I would add return types for all of these (same for service).

Right now for `findAll` you are returning `User` type that comes from `import { User } from '@prisma/client';`. It's better to separate database model / entity from presentation model. It's not a blocker for this PR (can be done in the next one), but I would add a `dto` folder where I would define requests and response DTO classes (here we need just a response DTO class). This way we have more control over what data we are returning from what endpoint.

Also note that some tutorials out there define DTOs that are shared between controllers / routes. I'm personally not a fan of this and prefer to have a separate DTO for each route. Although it's more duplicated code this way, we can later freely change these DTOs separately without affecting others.